src/request.ts

```diff
@@ -1119,11 +1126,59 @@ class DatastoreRequest {
         client: 'DatastoreClient',
         method: 'commit',
         reqOpts,
-        gaxOpts: gaxOptions || {},
+        gaxOpts: gaxOptions,
       },
       onCommit
     );
   }
+  /**
+   * Find the properties which value size is large than 1500 bytes,
+   * with autoUnIndex enable, automatically exclude properties from indexing.
+   * This will allow storing string values larger than 1500 bytes
+   *
+   * @param entities Datastore key object(s).
+   * @param path namespace of provided entity properties
+   * @param properties properties which value size is large than 1500 bytes
+   */
+  private findLargeProperties_(
```

Comment:
Do you think this could live inside of the `entity.ts` file, since other similar things are there related to handling and transforming entity objects?

Reply:
done!